Integrations/integration-PhishMe.yml

```diff
@@ -34,6 +34,30 @@ configuration:
   defaultvalue: "false"
   type: 8
   required: false
+- display: URL Threshold (None, Minor, Moderate or Major). Minimum severity to consider
+    the URL malicious
+  name: urlThreshold
+  defaultvalue: Major
+  type: 0
```

Comment:
change type to multi select with the relevant values (None,Minor,Moderate,Major)